capz/run-capz-e2e.sh

```diff
@@ -33,6 +33,11 @@ main() {
     export CAPI_VERSION="${CAPI_VERSION:-"v1.7.2"}"
     export HELM_VERSION=v3.14.4
     export TOOLS_BIN_DIR="${TOOLS_BIN_DIR:-$SCRIPT_ROOT/tools/bin}"
+    export WORKLOAD_IDENTITY_ID="${WORKLOAD_IDENTITY_ID:-"34d8e06d-d198-477e-b166-6936e58d90ae"}"
+    export AZURE_SUBSCRIPTION_ID="${AZURE_SUBSCRIPTION_ID:-"46678f10-4bbb-447e-98e8-d2829589f2d8"}"
+    export AZURE_TENANT_ID="${AZURE_TENANT_ID:-"d1aa7522-0959-442e-80ee-8c4f7fb4c184"}"
+    export CLIENT_ID="${CLIENT_ID:-"34d8e06d-d198-477e-b166-6936e58d90ae"}"
+    export REGISTRY="${REGISTRY:-"capzcicommunity.azurecr.io"}"
```

Comment:
for now, lets do 
```suggestion
    export REGISTRY=""capzcicommunity.azurecr.io"
```

since https://github.com/kubernetes/test-infra/pull/32937 the preset will override this